docroot/themes/custom/twentyeighteen/sass/components/call-to-action/call-to-action.twig

```diff
@@ -0,0 +1,9 @@
+<div class= "call-to-action-block">
+  <div class= "call-to-action-img">
+  <img src="https://static.pexels.com/photos/59919/pexels-photo-59919.jpeg">
+  </div>
+  <div class= "call-to-action-text">
+  <h1>Connect</h1>
```

Comment:
Once this is in Drupal, it will most likely be rendered as an h2. Can we update it in the twig file as well as in the sass?